include/gatekeeper_gk.h

```diff
@@ -19,18 +19,23 @@
 #ifndef _GATEKEEPER_GK_H_
 #define _GATEKEEPER_GK_H_
 
+#include <rte_ring.h>
 #include <rte_atomic.h>
+#include <rte_mempool.h>
 
 #include "gatekeeper_ipip.h"
+#include "gatekeeper_ggu.h"
 
 /* Structures for each GK instance. */
 struct gk_instance {
-	struct rte_hash   *ip_flow_hash_table;
-	struct flow_entry *ip_flow_entry_table;
+	struct rte_hash    *ip_flow_hash_table;
+	struct flow_entry  *ip_flow_entry_table;
```

Comment:
The DPDK style guide says to use spaces for alignment in structs, except for very long entries.